.env.local

```diff
@@ -0,0 +1,2 @@
+VITE_API_URL=http://localhost:3000/api
```

Comment:
why do we need these 2 variables?

Reply:
the first one is for making calls to the backend with out hardcoding the url and the second defines the mode of the app like development or production for example

Reply:
I doubt we need these variables, we even don't use them